extensions/display-modes/src/index.tsx

```diff
@@ -0,0 +1,115 @@
+import { ActionPanel, List, Action, useNavigation, showHUD, KeyEquivalent } from "@raycast/api";
+import { ReactNode, useEffect, useState } from "react";
+import { listDisplays, numberToKey, setMode } from "./utils";
+import { DisplayInfo, DisplayKind, Mode, areModesEqual } from "./types";
+
+export default function Command() {
+  const [displays, setDisplays] = useState<DisplayInfo[] | undefined>();
+
+  async function fetchDisplaysInfo() {
+    const displays = await listDisplays();
+    setDisplays(displays);
+  }
+
+  useEffect(() => {
+    fetchDisplaysInfo();
+  }, []);
+
+  function formatTitle(display: DisplayInfo): string {
+    return `Display ${display.display.id} (${formatDisplayKind(display.display.kind)})`;
+  }
+
+  function formatSubtitle(display: DisplayInfo): string {
+    return formatDisplayMode(display.currentMode);
+  }
+
+  function formatDisplayKind(kind: DisplayKind): string {
+    switch (kind) {
+      case DisplayKind.builtIn:
+        return "Built-in";
+      case DisplayKind.external:
+        return "External";
+      default:
+        return "Unknown";
+    }
+  }
+
+  function formatDisplayMode(mode: Mode): string {
+    const widthFormatted = mode.width.toString().padStart(5, " ");
+    const heightFormatted = mode.height.toString().padStart(5, " ");
+    const scaleFormatted = mode.scale?.toString().padStart(2, " ") ?? "N/A";
+    const refreshRateFormatted = mode.refreshRate.toString().padStart(4, " ");
+
+    return `${widthFormatted} x ${heightFormatted} @ ${scaleFormatted}x ${refreshRateFormatted}Hz`;
+  }
+
+  const navigation = useNavigation();
+
+  function detail(display: DisplayInfo): ReactNode {
+    // First, map the modes to an array of objects with mode and isEqual properties
+    const modesWithComparison = display.modes.map((mode) => ({
+      mode: mode,
+      isEqual: areModesEqual(mode, display.currentMode),
+    }));
+
+    return (
+      <List navigationTitle={formatTitle(display)}>
+        {modesWithComparison.map((modeInfo, index) => {
+          const key = numberToKey(index + 1);
+          return (
+            <List.Item
+              key={index}
+              icon={modeInfo.isEqual ? "selected-mode-icon.png" : "mode-icon.png"}
+              title={formatDisplayMode(modeInfo.mode)}
+              subtitle={modeInfo.isEqual ? "Current Mode" : undefined}
+              actions={
+                <ActionPanel>
+                  <Action
+                    title="Change Display Mode"
```

Comment:
```suggestion
                    title="Change Display Mode"
                    icon={Icon.Monitor}
```
Adding an icon makes it even more beautiful 🙂

Reply:
Done. Looks better!